demo/plugins.txt

```diff
@@ -3,33 +3,35 @@ branch-api:1.10
 cloudbees-folder:5.12
 credentials:2.1.4
 durable-task:1.12
-git:2.5.2
-git-client:1.19.7
+git:2.5.3
+git-client:1.21.0
 git-server:1.7
 handlebars:1.1.1
 icon-shim:2.0.3
 jquery-detached:1.2.1
-junit:1.15
+junit:1.18
+lockable-resources:1.10
 mailer:1.17
 matrix-project:1.7.1
 mock-slave:1.9
 momentjs:1.1.1
 pipeline-build-step:2.2
-pipeline-input-step:2.0
-pipeline-rest-api:1.6
-pipeline-stage-step:2.1
-pipeline-stage-view:1.6
+pipeline-graph-analysis:1.1
+pipeline-input-step:2.1
+pipeline-rest-api:2.0
+pipeline-stage-step:2.2
+pipeline-stage-view:2.0
 scm-api:1.2
-script-security:1.21
+script-security:1.22
 ssh-credentials:1.12
-structs:1.3
+structs:1.4
```

Comment:
Note that I just released 1.5 but it is not yet on the UC. The bug in 1.4 does not seem to affect this demo anyway.